model/host/host_intent.go

```diff
@@ -9,6 +9,8 @@ import (
 	"github.com/pkg/errors"
 )
 
+const maxImagesPerParent = 3
```

Comment:
can we make this configurable at the distro level?

Reply:
Added a pool setting to control this